docs/en/operations/settings/index.md

```diff
@@ -7,90 +7,16 @@ pagination_next: en/operations/settings/settings
 
 # Settings Overview
 
-There are multiple ways to define ClickHouse settings. Settings are configured in layers, and each subsequent layer redefines the previous values of a setting.
+There are two main groups of ClickHouse settings:
 
-The order of priority for defining a setting is:
+- Global server settings
```

Comment:
There are two main groups of ClickHouse settings:
- Global server settings
- Session and query-level settings